grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -145,10 +153,22 @@ class Order implements Serializable {
     }
 
     def totalPrice() {
-        def totalPrice = orderItems.collect { it.totalPrice() }.sum()
-        return totalPrice ?: 0
+        return orderItems.collect { it.totalPrice() }.sum() ?: 0
     }
 
+    def getTotalAdjustments() {
+        return orderAdjustments?.sum {
+            return it.amount ?: it.percentage ? (it.percentage/100) * totalPrice() : 0
+        }?:0
+    }
+
+    def getSubtotal() {
+        totalPrice()
+    }
+
+    def getTotal() {
```

Comment:
getTotal() and totalPrice() naming here is missleading. I think it should be more clear.

Reply:
I didn't finish refactoring because I didn't want to break a bunch of pages that reference totalPrice(). But I rewrote the code slightly to make that more obvious and now totalPrice just references the getTotal() method.